src/main/java/com/commercetools/sync/products/ProductSyncOptions.java

```diff
@@ -0,0 +1,37 @@
+package com.commercetools.sync.products;
+
+import com.commercetools.sync.commons.helpers.BaseSyncOptions;
+import io.sphere.sdk.commands.UpdateAction;
+import io.sphere.sdk.products.Product;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class ProductSyncOptions extends BaseSyncOptions {
```

Comment:
https://github.com/commercetools/commercetools-sync-java/pull/10/commits/489721531dd774babf0b9fc3d4ca8383a99d0d8b I need to keep it as a skeleton for when the product sync implementation starts.